src/locale/en.json

```
@@ -290,19 +290,19 @@
     "productViewer": {
       "actions": {
         "compareButton": "Compare",
-        "toAasView": "AAS View"
+        "toAasView": "AAS view"
       },
       "classification": "Classification",
       "error": {
-        "technicalDataSubmodelNotFound": "Technical Data Submodel not found"
+        "technicalDataSubmodelNotFound": "Technical data submodel not found"
```

I would say that 'Technical Data Submodel' is an Eigenname (name on it's own??) and should be capitalized
```suggestion
        "technicalDataSubmodelNotFound": "Technical Data Submodel not found"
```

---

Solved